.pylintrc

```diff
@@ -0,0 +1,50 @@
+# Note:
+# This is a minimal subset of the options available for Pylint.
+# To generate an exhaustive configuration file based on current settings and defaults, run the following command
+# in the project directory:
+#
+#  pylint --generate-rcfile > ~/.pylintrc
+
+[MASTER]
+
+# Add files or directories to the blacklist. They should be base names, not
+# paths.
+ignore=.git
+
+# Directories from top level that should be ignored.
+ignore-paths=neuro_san/api/grpc
+
+# Add files or directories matching the regex patterns to the blacklist. The
+# regex matches against base names, not paths.
+ignore-patterns=
+
+# Use multiple processes to speed up Pylint. Specifying 0 will auto-detect the
+# number of processors available to use.
+jobs=0
+
+# Set reasonable line length
+max-line-length=120
+
+# Disable the message, report, category or checker with the given id(s). You
+# can either give multiple identifiers separated by comma (,) or put this
+# option multiple times (only on the command line, not in the configuration
+# file where it should appear only once).You can also use "--disable=all" to
+# disable everything first and then reenable specific checks. For example, if
+# you want to run only the similarities checker, you can use "--disable=all
+# --enable=similarities". If you want to run only the classes checker, but have
+# no Warning level messages displayed, use"--disable=all --enable=classes
+# --disable=W"
+
+disable=
+    duplicate-code,
+    missing-module-docstring,
+    missing-function-docstring,
+    logging-fstring-interpolation,
+
+# Work to remove these
+max-args=15
+max-attributes=16
+max-branches=20
+max-locals=30
+max-positional-arguments=15
+max-statements=75
```

Comment:
More non-standard limits that reduces the signal-to-noise ratio of complaints quickly.
Can work to lower/remove these later.